### PR TITLE
Blog post on our recommendation to avoid Commons Clause

### PR DESCRIPTION
**Pull request by baggerspion**
## Description

Adding a blog post on our rationale for recommending that teams avoid Commons Clause licensed code.

## Types of Changes

_What types of changes does your code introduce? remove the points which aren't applicable:_

- New blog post

## Tasks

  - [x] Draft outline for review
  - [ ] Full draft for review
  - [ ] Add header image and publish

### CODE REVIEW

File: _posts/2018-12-03-commons-clause.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Commons Clause at Zalando
```
Comment (perploug) [inline _posts/2018-12-03-commons-clause.md:2]: I think this should both address Commons Clause and Server-Side Public License

File: _posts/2018-12-03-commons-clause.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+# Introduction
+- What is the CC license?
+- Where/why was it created?
+
+# The Spirit of Open Source
+- The importance of Freedom 0
+- When you commit to Open Source, you commit to your community
+- Commons Clause breaks this covenant
```
Comment (perploug) [inline _posts/2018-12-03-commons-clause.md]: Mongo and Redis have benefitted from open source and community contributions - and now they cash in - would anyone have contributed to this if the commons clause was in place before..

File: _posts/2018-12-03-commons-clause.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+# The Problems of Monetisation
+- Can't sell something that is Free/free
+- So what can we sell?
+  - Consulting
+  - Feature development
+  - Training
+  - Lots of things!
+- It can be done! Look at RedHat!
+
+# Taking A Stand Against Commons Clause
```
Comment (perploug) [inline _posts/2018-12-03-commons-clause.md]: Redis Labs Modules already forked: https://www.zdnet.com/article/redis-labs-and-common-clause-attacked-where-it-hurts-with-open-source-code/

**Review by perploug — COMMENTED** [review]
couple of things I would include - most importantly, including the ServerSide Public License - and btw - why does both of these licenses try to sound like they are all open and free and lalalala ...

File: _posts/2018-12-03-commons-clause.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
```
Comment (baggerspion) [inline _posts/2018-12-03-commons-clause.md]: Should talk here about: the clause vs guidance.

**Review by hpdang — APPROVED** [review]

**Pull request closed without merging**